src/sys/socket/sockopt.rs

```diff
@@ -1015,6 +1039,50 @@ sockopt_impl!(
     libc::IP_TTL,
     libc::c_int
 );
+#[cfg(any(apple_targets, linux_android))]
+#[cfg(feature = "net")]
+sockopt_impl!(
+    /// Retrieve the current time-to-live field for every
+    /// IPv4 packet received on this socket.
+    IpRecvTtl,
+    Both,
+    libc::IPPROTO_IP,
+    libc::IP_RECVTTL,
+    bool
+);
```

Comment:
From the [man](https://man.freebsd.org/cgi/man.cgi?query=ip&apropos=0&sektion=4&manpath=FreeBSD+14.0-RELEASE&arch=default&format=html) page, this option is also available on FreeBSD.

Reply:
`libc` doesn't seem to export it: https://github.com/nix-rust/nix/pull/2203/checks?check_run_id=20299209049

Reply:
I [tried to add these 2 constants to the libc crate](https://github.com/rust-lang/libc/pull/3543), though I have no idea about the CI failure...

Reply:
You can also adjust our libc git dependency and enable this option for FreeBSD as that libc PR is merged